### PR TITLE
[CIAM-1035] IBM Semeru JDK is not installed properly. Also remove JDK 8 since not needed anymore

### PR DESCRIPTION
**Pull request by drichtarik**
@iankko @levivic

### CODE REVIEW

File: modules/sso/sso-rm-openjdk/11/configure.sh [hunk 1]
```diff
@@ -2,19 +2,8 @@
...
-           java-1.8.0-openjdk; do
-        if rpm -q "$pkg"; then
-            rpm -e --nodeps "$pkg"
-        fi
-    done
-fi
-
 ## Work around OpenJDK being installed as dependency. https://bugzilla.redhat.com/show_bug.cgi?id=1762827 and similar
-if rpm -q java-11-openj9-devel; then
+if rpm -q ibm-semeru-open-11-jdk; then
```
Comment (iankko) [inline modules/sso/sso-rm-openjdk/11/configure.sh]: @drichtarik I would probably keep this covering both cases. IOW to remove `openjdk-devel` if either `ibm-semeru-open-11-jdk` or `java-11-openj9-devel` are present.

Though `java-11-openj9-devel` can't be installed at the moment, this is just convenience approach for the future, if / once we would switch the OpenJ9 overrides module to start installing `java-11-openj9-devel` again.

@levivic -^ WDYT should we cover both possibilities?

**Review by iankko — COMMENTED** [review]
@drichtarik Thanks for the change. One small nitpick (to cover both OpenJ9 & IBM Semeru cases in `sso-rm-jdk` module) yet, but rest of the changes LGTM.

**Review by iankko — APPROVED** [review]
The latest version LGTM. Thx Denis!

File: modules/sso/sso-rm-openjdk/11/configure.sh (hunk 1 above)
Comment (iankko) [reply modules/sso/sso-rm-openjdk/11/configure.sh]: Checking for both variants is covered in latest patch form.

**Comment by drichtarik** [comment]
Run 234 was finally successful. @levivic told me he will let us know if the correct java was used. I provided him with the scratch build done by this PR.

**Review by iankko — CHANGES_REQUESTED** [review]
See @levivic 's comment above. @drichtarik That openjdk module installation needs to be removed from the eap module definition.

**Comment by drichtarik** [comment]
@iankko run 263 was successful with the latest update

**Comment by levivic** [comment]
@drichtarik  this looks good to me.

**Pull request merged**